client/src/views/ScorecardView.vue

```diff
@@ -42,6 +42,11 @@
         </div>
       </div>
 
+      <ContactCitySection
```

Comment:
Is there a reason this isn't in the section above, which has the other CTAs?

Reply:
I tried putting it inside of that div as another 1/3 of the page element but it looked really strange: https://screenshot.googleplex.com/9Ft8xw6DsXcwL5J I think because visually it is different from those other ctas so it looks strange to me that they are grouped in a row like that (as opposed to https://screenshot.googleplex.com/6bsiMhQtph5iZzg). 

Let me know what you think!